config/index.js

```diff
@@ -14,6 +14,7 @@ const {
   APP_ID,
   APPLICATION_NAME,
   EXPLORER_URL,
+  NOTIFICATION_EMAIL_URL,
```

Comment:
unused for now, should be exported. Please don't give a generic name like `baseURL` because there will be other servers URLs in the future, for example for airdrop

Reply:
why it is name EMAIL? should be NOTIFICATION_API_URL or something like that

Reply:
I agree with that. The name of the variable should say it's about API